src/heads-up/components/app-sidebar.vue

```diff
@@ -36,6 +36,12 @@
         </li>
       </ul>
     </section>
+    <footer class="sidebar__footer">
+      Made with love by 
+      <external-link class="sidebar__footer-anchor" href="https://www.voorhoede.nl" target="_blank">
```

Comment:
Can you remove `target="_blank"`? This is already used in the `<external-link>` component.